### PR TITLE
Feature - Add sortable table components

### PR DESCRIPTION
**Pull request by ShahStavan**

### CODE REVIEW

**Comment by ShahStavan** [comment]
Hi @markmead 
All changes you asked for are done. Please review the code.
Thank You.

**Review by markmead — CHANGES_REQUESTED** [review]
Shouldn't be `dark:` classes, or `id` attributes, or `onclick` events.

File: public/components/application-ui-tables/8.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            Date of Birth 
+            <span class="sort-icon">&uarr;&darr;</span>
+        </th>
+        <th class="text-left cursor-pointer whitespace-nowrap px-4 py-2 font-medium text-gray-900">
+            Role 
+            <span class="sort-icon">&uarr;&darr;</span>
+        </th>
+        <th class="p-2 text-left cursor-pointer whitespace-nowrap px-4 py-2 font-medium text-gray-900">
+            Salary 
+            <span class="sort-icon">&uarr;&darr;</span>
```
Comment (markmead) [inline public/components/application-ui-tables/8.html:19]: I think we should use a chevron here:

```
<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="size-6">
  <path stroke-linecap="round" stroke-linejoin="round" d="m19.5 8.25-7.5 7.5-7.5-7.5" />
</svg>
```

Might need some tweaking as `size-6` is pretty large.

Comment (markmead) [reply public/components/application-ui-tables/8.html:19]: No need for the `sort-icon` class either.

File: src/data/components/application-ui-tables.mdx [hunk 2]
```diff
@@ -29,6 +29,10 @@ components:
   7:
     title: Bordered with divided rows and pagination
     dark: true
+  8:
+    title: Bordered with divided rows and sorting
+    dark: false
```
Comment (markmead) [inline src/data/components/application-ui-tables.mdx:34]: Can remove this line.

**Review by markmead — COMMENTED** [review]
Looks good! Just a few comments to address and then we can start the dark and interactive variants! 🎉

**Comment by markmead** [comment]
Are you still working on this? @ShahStavan Or do you want me to take a look?

**Pull request closed without merging**